infra/scripts/start-server.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -e
+
+source activate TEST;
+./infra/scripts/wait-for-postgres.sh python backend/server/manage.py migrate;
```

Comment:
Umm... don't you need a `;` after `./infra/scripts/wait-for-postgres.sh`?

Reply:
You don't have to, but yes it bothers me that that thing has no semicolon lmao

Reply:
No you don't haha